app/javascript/components/applications/application_form/formOptions.js

```diff
@@ -1,21 +1,30 @@
 import _ from 'lodash'
 
 const labelize = (options) => {
-  if (!_.isEmpty(options)) {
-    let initialLabel = []
-    if ((options[0].hasOwnProperty('value') && !_.isEmpty(options[0].value)) || (!options[0].hasOwnProperty('value') && !_.isEmpty(options[0]))) {
-      initialLabel = [{'label': 'Select One...', 'value': ''}]
-    }
-    return initialLabel.concat(
-      _.map(options, (option) => (
-        { value: option.hasOwnProperty('value') ? option.value : option, label: option.label || option }
-      ))
-    )
-  } else {
-    return []
+  if (_.isEmpty(options)) return []
+
+  let emptyInitialOptionPresent =
+    _.isEmpty(options[0]) ||
+    (options[0].hasOwnProperty('value') && _.isEmpty(options[0].value))
+
+  let labelizedOptions = []
+  if (!emptyInitialOptionPresent) {
+    labelizedOptions.push({'value': '', 'label': 'Select One...'})
   }
+
+  return labelizedOptions.concat(
+    _.map(options, (option) => (
+      {
+        value: option.hasOwnProperty('value') ? option.value : option,
+        label: option.label || option
+      }
+    ))
+  )
 }
 
+// TODO: Remove the use of labelize on all the below options arrays once the
```

Comment:
Do we need to create a chore for this so we don't forget it?

Reply:
You could add it as an AC/task on the Pivotal story if you feel that would suffice.

Reply:
added! https://www.pivotaltracker.com/story/show/165735996